<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<node name="goal_reached" pkg="ego_planner" type="goal_reached.py" output="screen">
	</node>

</launch>